### PR TITLE
Update surfacehub-csp.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2108

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/surfacehub-csp.md [hunk 1]
```diff
@@ -228,15 +228,15 @@ The following diagram shows the SurfaceHub CSP management objects in tree format
...
+<p style="margin-left: 20px">The data type is string. Supported operation is Get and Replace.
 
 <a href="" id="inboxapps-welcome"></a>**InBoxApps/Welcome**  
 <p style="margin-left: 20px">Node for the welcome screen.
 
 <a href="" id="inboxapps-welcome-autowakescreen"></a>**InBoxApps/Welcome/AutoWakeScreen**  
 <p style="margin-left: 20px">Automatically turn on the screen using motion sensors.
 
-<p style="margin-left: 20px">The data type is bool. Supported operation is Get and Replace.
+<p style="margin-left: 20px">The data type is booleab. Supported operation is Get and Replace.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/surfacehub-csp.md]:
```suggestion
<p style="margin-left: 20px">The data type is boolean. Supported operation is Get and Replace.
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 , your recommendation was successfully applied, thank you!

**Comment by mypil** [comment]
@v-madhi - Can you please assist to check, approve and merge the content of this PR based on issue #2108?

Thank you.

**Pull request merged**